<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_routine</class>
 <widget class="QWidget" name="new_routine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>65</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
           background-color: #FFFFF0;
           color: #0d172a;
           border: 0px solid black;
           border-radius: 10px;
           padding: 5px;
           margin: 3px;
}
QLineEdit{
	border:2px solid rgb(59, 59, 59);
	border-radius: 10px;
	font-size: 15px;
  	font-weight: 600;
  	line-height: 1;
  	padding: 2px 2px;
	background:white;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="period_no">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 15px;
font-weight: 600;
line-height: 1;
padding: 2px 2px;
color: black;
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="class_name">
     <property name="maxLength">
      <number>21</number>
     </property>
     <property name="placeholderText">
      <string> Class name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="instructor">
     <property name="maxLength">
      <number>15</number>
     </property>
     <property name="placeholderText">
      <string> Instructor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="class_location">
     <property name="maxLength">
      <number>15</number>
     </property>
     <property name="placeholderText">
      <string> Building+Room</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
